autoload/clap/themes/srcery.vim

```diff
@@ -0,0 +1,137 @@
+" Author: Birger J. Nordolum  <contact+srcery@mindtooth.no>
+" Description: Clap theme based on the Srcery color scheme.  The theme
+" require some tweaks to make it work.  If you are to copy this code for
+" your own color scheme, or build upon it.  Make sure that you are aware
+" of the additional tweaks for Srcery.
+
+
+let s:save_cpo = &cpoptions
+set cpoptions&vim
+
+
+" Helper Functions {{{
+
+function! s:GetHightlightColor(group, type) abort
+	if a:type ==? 'fg'
+		let l:gui_color = synIDattr(hlID(a:group), 'fg', 'gui')
+		let l:term_color = synIDattr(hlID(a:group), 'fg', 'cterm')
+	elseif a:type ==? 'bg'
+		let l:gui_color = synIDattr(hlID(a:group), 'bg', 'gui')
+		let l:term_color = synIDattr(hlID(a:group), 'bg', 'cterm')
+	endif
+
+	return [ l:gui_color, l:term_color ]
+endfunction
+
+" }}}
+" Srcery Palette {{{
+
+let s:black          = ['#1C1B19', 0]
+let s:red            = ['#EF2F27', 1]
+let s:green          = ['#519F50', 2]
+let s:yellow         = ['#FBB829', 3]
+let s:blue           = ['#2C78BF', 4]
+let s:magenta        = ['#E02C6D', 5]
+let s:cyan           = ['#0AAEB3', 6]
+let s:white          = ['#D0BFA1', 7]
+let s:bright_black   = ['#918175', 8]
+let s:bright_red     = ['#F75341', 9]
+let s:bright_green   = ['#98BC37', 10]
+let s:bright_yellow  = ['#FED06E', 11]
+let s:bright_blue    = ['#68A8E4', 12]
+let s:bright_magenta = ['#FF5C8F', 13]
+let s:bright_cyan    = ['#53FDE9', 14]
+let s:bright_white   = ['#FCE8C3', 15]
+
+" xterm colors.
+let s:orange        = ['#FF5F00', 202]
+let s:bright_orange = ['#FF8700', 208]
+let s:hard_black    = ['#121212', 233]
+let s:xgray1        = ['#262626', 235]
+let s:xgray2        = ['#303030', 236]
+let s:xgray3        = ['#3A3A3A', 237]
+let s:xgray4        = ['#444444', 238]
+let s:xgray5        = ['#4E4E4E', 239]
+let s:xgray6        = ['#585858', 240]
+
+"}}}
+" Srcery Tweaks {{{
+
+" The following tweaks is done to make sure that the theme works
+" properly. Because of reasons, we require that some additional actions
```

Comment:
Maybe be clearer on why you made the tweaks you did, not say 
> because of reasons
 
but something like
> because srcery has a transparent background as an option, some additional actions were taken to ensure that the clap colors look right regardless of this setting.

It isn't very clear why you did what you did.

Reply:
I'm a bit curious, is there other reasons why these tweaks are needed? does clap need SignColumn to be none?

Reply:
Either NONE or the color of the preview windows to make it take on the same color.  Somehow, clap reads a color and sets it accordingly, but in our case, it's the wrong color.

Look in this picture and you'll see the far left column being black instead of grayish :https://github.com/srcery-colors/srcery-vim/pull/48#issuecomment-670768752

I'll attack the rest now. 😉

Reply:
Right right, I remember you showing me. Glad you got it sorted :)